src/_includes/layouts/post.njk

```diff
@@ -1,19 +1,29 @@
+---js
+{
+    layout: "layouts/base.njk",
+    type: "post",
+    eleventyComputed: {
+        permalink: data => isFuturePost(data.date) ? false : `${data.page.filePathStem}/`,
+        eleventyExcludeFromCollections: data => isFuturePost(data.date) ? true : false
```

Comment:
If you're method name, or Filter in this case, is `Post` related. Why is the only argument a DateTime Object? I don't think it matters to pass the whole object as you're passing a reference.

Futher, if the method starts with `isXYZ` I expect a boolean to be returned, so can we make this:

```suggestion
        eleventyExcludeFromCollections: data => isFuturePost(data)
```